### PR TITLE
Add Deno to getting started

### PR DESCRIPTION
**Pull request by anishkny**

### CODE REVIEW

File: getting-started/developers-guide.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ layout: getting-started
   - [from JavaScript code](https://developer.mozilla.org/en-US/docs/WebAssembly/Loading_and_running)
   - [as a CLI application](https://github.com/bytecodealliance/wasmtime/blob/master/docs/WASI-tutorial.md)
   - [from Node.js (with access to system resources)](https://nodejs.org/api/wasi.html)
+  - [from Deno](https://deno.land/manual/getting_started/webassembly)
```
Comment (RReverser) [inline getting-started/developers-guide.md:28]: Link is invalid.

**Review by RReverser — CHANGES_REQUESTED** [review]

File: getting-started/developers-guide.md (hunk 1 above)
Comment (anishkny) [reply getting-started/developers-guide.md]: Whoops sorry about that! Thanks for checking. Should be fixed now.

**Comment by RReverser** [comment]
Looking at the updated (working) link, it doesn't look like Deno is doing anything special here that isn't already covered by "from JavaScript code"? We have custom section for Node.js only because of WASI support, but for generic JS code it doesn't make sense to list each engine separately.

**Comment by RReverser** [comment]
Going to close this for now, but happy to reopen if my understanding is incorrect.

**Pull request closed without merging**